exercises/D-logical-operators/exercise2.js

```diff
@@ -5,8 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
```

Comment:
While not wrong,
```javascript
if (someCondition) {
    return true;
} else {
    return false;
}
```
Is equivalent to
```bash
return someCondition;
```

In this case, you could write:
```bash
function isNegative(num) {
    return num < 0;
}
```

As I've said, what you wrote it's okay, just good to be aware of both :)